build/conda/installer/assets/create_shortcut.vbs

```diff
@@ -1,16 +1,53 @@
 Set objShell = CreateObject("WScript.Shell")
-sDesktopPath = objShell.SpecialFolders("Desktop")
+sdesktopPath = objShell.SpecialFolders("Desktop")
+shortcutFolder = sdesktopPath & "\OpenBB Shortcuts"
 
-' Create shortcut for openbb-cli
-Set objShortcut = objShell.CreateShortcut(sDesktopPath & "\openbb-cli.lnk")
-objShortcut.TargetPath = objShell.ExpandEnvironmentStrings("%USERPROFILE%") & "\openbb\Scripts\openbb-cli.exe"
-objShortcut.IconLocation = objShell.ExpandEnvironmentStrings("%USERPROFILE%") & "\openbb\assets\openbb.ico"
-objShortcut.Description = "OpenBB CLI launcher"
+Set objFSO = CreateObject("Scripting.FileSystemObject")
+If Not objFSO.FolderExists(shortcutFolder) Then
+    objFSO.CreateFolder(shortcutFolder)
+End If
+
+prefixPath = objShell.ExpandEnvironmentStrings("%PREFIX%")
+userProfilePath = objShell.ExpandEnvironmentStrings("%USERPROFILE%")
+
+If prefixPath = "%PREFIX%" Or userProfilePath = "%USERPROFILE%" Then
+    WScript.Echo "Environment variables %PREFIX% or %USERPROFILE% are not set correctly."
+    WScript.Quit 1
+End If
+
+Sub CreateShortcut(name, targetPath, iconPath)
+    Set objShell = CreateObject("WScript.Shell")
+    Set objShortcut = objShell.CreateShortcut(shortcutFolder & "\" & name & ".lnk")
+    objShortcut.TargetPath = objShell.ExpandEnvironmentStrings(targetPath)
+    objShortcut.IconLocation = objShell.ExpandEnvironmentStrings(iconPath)
+    objShortcut.Description = name
+    objShortcut.Save
+End Sub
+
+CreateShortcut "OpenBB CLI", prefixPath & "\Scripts\openbb-cli.exe", prefixPath & "\assets\openbb_icon.ico"
```

Comment:
I think we can ask Rita to create distinct icons for all these apps. Or I can do that. What is the full list, can you reply to this comment please, so that I grab it?

Reply:
"OpenBB API"
"OpenBB Notebook"
"OpenBB IPython"
"OpenBB Updater"
"OpenBB CMD" -> Windows-only
"OpenBB Bash" -> Unix-only
"OpenBB User Settings"
"OpenBB User Data"
"Uninstall" -> Windows-only